new_sem3/dz2_no_duplicates_list.py

```diff
@@ -1,3 +1,14 @@
 # Дан список повторяющихся элементов.
 # Вернуть список с дублирующимися элементами.
 # В результирующем списке не должно быть дубликатов.
+
+numbers = [11, 2, 3, 3, 2, 3, 4, 5]
+print(f'{numbers = }')
+
+res = []
+count = 0
+for value in numbers:
+    if numbers.count(value) > 1 and value not in res:
```

Comment:
С точки зрения сложности так будет чуть лучше
```suggestion
for value in set(numbers):
    if numbers.count(value) > 1:
```